website/docs/layouts/grid/partials/code/how-to-use.md

```diff
@@ -0,0 +1,284 @@
+
+##### Grid within a wider view
+
+Narrow your browser window to see the responsive behavior.
+
+```handlebars
+<Hds::Layout::Grid @columnMinWidth="160px" @gap="16">
+  <Doc::Placeholder @height="40px" @text="Item 1" @background="#e4e4e4" />
+  <Doc::Placeholder @height="40px" @text="Item 2" @background="#e4e4e4" />
+  <Doc::Placeholder @height="40px" @text="Item 3" @background="#e4e4e4" />
+  <Doc::Placeholder @height="40px" @text="Item 4" @background="#e4e4e4" />
+</Hds::Layout::Grid>
+```
+
+##### The same grid within a narrower view
+
+At the specified column min width, columns are forced to stack in this narrower view.
+
+```handlebars
+<div class="doc-grid-mobile-view">
+  <Hds::Layout::Grid @columnMinWidth="160px" @gap="16">
+    <Doc::Placeholder @height="40px" @text="Item 1" @background="#e4e4e4" />
+    <Doc::Placeholder @height="40px" @text="Item 2" @background="#e4e4e4" />
+    <Doc::Placeholder @height="40px" @text="Item 3" @background="#e4e4e4" />
+    <Doc::Placeholder @height="40px" @text="Item 4" @background="#e4e4e4" />
+  </Hds::Layout::Grid>
+</div>
+```
+
+### Align
+
+Align grid items to the "start", "end", "center" or "stretch" them within the grid parent.
+
+Note: The `Grid` parent will need a height set for the affect to be visible.
+
+```handlebars
+<div class="doc-grid-mobile-view">
+  <Hds::Layout::Grid @columnMinWidth="50px" @gap="16" @align="center" {{style height="100%"}}>
+    <Doc::Placeholder @height="40px" @text="Item 1" @background="#e4e4e4" />
+    <Doc::Placeholder @height="40px" @text="Item 2" @background="#e4e4e4" />
+    <Doc::Placeholder @height="40px" @text="Item 3" @background="#e4e4e4" />
+    <Doc::Placeholder @height="40px" @text="Item 4" @background="#e4e4e4" />
+  </Hds::Layout::Grid>
+</div>
+```
+
+### isInline
+
+To change the default display from `grid` to `inline-grid`, set `@isInline` to `true`.
+
+```handlebars
+Text before the inline grid.
+<Hds::Layout::Grid  @isInline={{true}} @tag="span" @gap="8">
+  <Doc::Placeholder @height="1.2em" @text="Item 1" @background="#e4e4e4" />
+  <Doc::Placeholder @height="1.2em" @text="Item 2" @background="#e4e4e4" />
+  <Doc::Placeholder @height="1.2em" @text="Item 3" @background="#e4e4e4" />
+  <Doc::Placeholder @height="1.2sem" @text="Item 4" @background="#e4e4e4" />
+</Hds::Layout::Grid>
+Text after the inline grid.
+```
+
+### Grid::Item
+
+The `Grid::Item` component can optionally be used to wrap `Grid` content if more control is needed over the grid layout.
+
+### colSpan & rowSpan
+
+Use the `colSpan` and `rowSpan` options of `Grid::Item` components to set the number of columns or rows an item should occupy.
+
+Note: By default, if a height is set on the `Grid` parent, row heights will stretch proportionally to fill the Grid. To instead make a row conform to the minimum height of its content, you can pass an inline style as shown in the example.
```

Comment:
I tried to think of a feature to solve this issue (of grid rows not sizing proportionally to content). So far, I haven't been able to come with a simple non-confusing API.

This really is only an issue for particular layouts in which a height is set for the `Grid` container though so I decided, for now, to just explain the issue and show how to fix it with an inline style in this example.

Reply:
The issue is with the underlying grid, not with the content. If I take out the `grid-template-rows` inline style it still does this:

<img width="726" alt="image" src="https://github.com/user-attachments/assets/7df0e490-dcb6-40da-afac-cabdb92b9de6" />

I'll remove the height=100% on the top placeholder though as that makes the example more confusing

Reply:
Ahh I see what you mean!